src/plugins/embeddable/public/react_embeddable_system/react_embeddable_renderer.tsx

```diff
@@ -33,12 +33,14 @@ export const ReactEmbeddableRenderer = <
   state,
   parentApi,
   onApiAvailable,
+  panelProps,
 }: {
   maybeId?: string;
   type: string;
   state: SerializedPanelState<StateType>;
   parentApi?: PresentationContainer;
   onApiAvailable?: (api: ApiType) => void;
+  panelProps?: Omit<PresentationPanelProps<ApiType>, 'Component'>;
```

Comment:
**nit** I wonder - would `Pick<PresentationPanelProps<ApiType>, 'showShadow' | 'showBorder'>` be cleaner? It looks like `PresentationPanelProps` contains a lot of other stuff (for example, `onPanelStatusChange`, `componentProps`, etc.) and it's possible more might be added.

We also use `Pick` rather than `Omit` when definining `PresentationPanelHeaderProps` so it's more consistent, too.

Reply:
I updated the type and included the other props related to panel visuals https://github.com/elastic/kibana/pull/178139/commits/6cf53421b7cab06844a0f848536e5a71e05b2db4